dist/zone.js.d.ts

```diff
@@ -5,6 +5,7 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+/// <amd-module name="__main__/lib/zone" />
```

Comment:
this is a bit gross, users don't want to see __main__ - in angular we do something to trim these lines from published .d.ts

Reply:
Got it, could you show me the source that how `angular` trim those lines? Thanks